### PR TITLE
EIP-1: clarify the process around core EIPs

### PR DESCRIPTION
**Pull request by axic**
This PR has two parts:
1) attempts to document the current process around Core EIPs a bit better
2) renames `Accepted` to `Ready`

On https://gitter.im/ethereum/EIPs there was a discussion about the `Accepted` state being misleading. Some new names were proposed: Reviewed, Stable, Live.

I think `Ready` is the best way to describe this status. It is ready to be reviewed by the "All Core Devs" and consider it for inclusion in a protocol update.

### CODE REVIEW

**Comment by axic** [comment]
Perhaps I should separate the renaming so that the clarification of EIP-1 wording can be merged sooner.

**Comment by fulldecent** [comment]
I fully support this PR and I am the last author of a lot of the lines that are being updated.

**Comment by axic** [comment]
@chriseth suggested `Complete` for the same state [here](https://github.com/ethereum/EIPs/pull/2147#discussion_r301554463), while @fubuloubu suggested introducing a new state `Review` prior to `Last Call` [here](#1725).

I'm leaning towards this PR is a good middle ground and close enough what happens right now, but clarifies the states better.

**Comment by axic** [comment]
@Arachnid @nicksavers @Souptacular @gcolvin any comments?

**Comment by axic** [comment]
Rebased after #2236.

File: _data/statuses.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 - Last Call
 - Draft
 - Accepted
```
Comment (nicksavers) [inline _data/statuses.yaml]: If *Ready* is the new *Accepted*, why leave the old status here?

Comment (axic) [reply _data/statuses.yaml]: I didn't update every affected EIP yet (and eip-validator needs an update too), so this was a temporary way to get this merged, then update tools and affected files, finally remove the old one.

Comment (axic) [reply _data/statuses.yaml]: I'd argue though it would be better for me to split the renaming into a new PR so that we can merge the clarification parts.

**Comment by axic** [comment]
@nicksavers split out the renaming to make this eaiser, we can argue about renaming and the process to do the renaming in #2279.

**Comment by Arachnid** [comment]
"Final" is the state where something is ready to be reviewed for inclusion by All Core Devs. Why are we introducing another status?

**Comment by axic** [comment]
> "Final" is the state where something is ready to be reviewed for inclusion by All Core Devs.

It's not. This is what it originally said:

>  Accepted (Core EIPs only) -- This EIP is in the hands of the Ethereum client developers. Their process for deciding whether to encode it into their clients as part of a hard fork is not part of the EIP process.
>
>    ➡️ Final -- Standards Track Core EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final. When the implementation is complete and adopted by the community, the status will be changed to “Final”.

It explicitly says that ACD looks at it at the Accepted stage and may consider moving it to the Final stage.

**Pull request merged**